src/models/thelio-spark-b1-n2/repairs.md

```diff
@@ -0,0 +1,630 @@
+
+**Tools required:** Cross-head (Phillips) screwdriver, thermal paste  
+**Time estimate:** 45 minutes  
+**Difficulty:** High <span style="color:red;">●</span>
+
+### Steps to remove the CPU cooler/thermal paste:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the side brace](#removing-the-side-brace), [remove the CPU duct](#removing-the-cpu-duct), and [remove the GPU](#replacing-the-gpu).
+    - Unplug the [heatsink-mounted CPU fan](#steps-to-replace-the-heatsink-mounted-cpu-fan) unless it's already been removed.
+2. While holding the CPU cooler in place so it doesn't fall, loosen the screws on the top and bottom of the CPU cooler holding it onto the mounting bracket.
+    - If you unscrew the screws too far, the 90-degree clips will detatch from the vertical mounting bar. Reattach them facing inwards when reinstalling the cooler.
+
+![CPU cooler screws](./img/cpu-cooler-screws.webp)
+
+3. Pull the cooler away from the CPU, alternating between the top and bottom to free the 90-degree clips from the mounting bracket.
+4. Using a paper towel, clean the existing thermal paste off of the heatsink and CPU. You may also use a small amount of rubbing alcohol if the old paste is dried or difficult to remove.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+### Steps to replace the CPU:
+
+1. Place the computer on its side so the motherboard is facing up.
+2. [Remove the RAM](#replacing-the-ram) to provide easier access to the cooler mounting bracket's plastic retention pins.
+3. Pull the plastic retention pins out of the four corners of the cooler mounting bracket, then remove the cooler mounting bracket.
+
+![CPU cooler mounting bracket](./img/cpu-cooler-mounting-bracket.webp)
+
+4. Push the locking pin outward until it's able to spring away from the motherboard.
+    - Caution: the locking pin may spring up with significant force when freed.
+
+![CPU locking pin](./img/cpu-locking-pin.webp)
+
+5. Flip the CPU holder away from the CPU.
+    - The CPU holder opens in the opposite direction from the locking pin.
+
+![CPU holder raised](./img/cpu-holder-raised.webp)
+
+6. Carefully lift the CPU out of the CPU socket.
+    - Be careful not to bend any of the gold pins on the CPU socket, and do not touch the gold pads on the CPU.
+
+![CPU (removed)](./img/cpu-removed.webp)
+
+7. Gently place the new CPU into the socket.
+    - The triangle on the CPU should be oriented to match the triangle on the CPU cover, pointing towards the bottom left of the motherboard.
+8. Flip the CPU cover back onto the CPU and push the locking pin down into place.
+9. Reinstall the cooler mounting bracket by lining it up with the holes in the motherboard and pushing the plastic retention pins back into place.
+    - Ensure that the transparent sleeves in the corners of the mounting bracket are pulled outwards to the `1700` position.
+    - The orientation of the cooler mounting bracket doesn't matter.
```

Comment:
The orientation is marked differently for LGA1200 and LGA1700 sockets on the bracket. Is it really the same either way? Seems weird that they'd bother marking it if it didn't matter.

![2023-11-14_09-21-56](https://github.com/system76/tech-docs/assets/13512097/347f68f8-4340-4a93-9292-c74cb3f4f7bb)

Reply:
Yes. I took the photos in the orientation that it was given to me in, but I initially put it back together rotated 90 degrees before I noticed those numbers in the photos. The bracket is square and the clips are the same size on every side; they just alternated the numbers on each side.

Reply:
I could add a "by default, they're this way" note. I just wasn't sure if that would make since given that the actual info here is that it doesn't matter.

Reply:
Huh, I'm surprised they bothered with the labels then. If it works either way, I don't think a note is necessary.

Reply:
It's to differentiate the bracket from the other ones that come in the box with the cooler. I went ahead and added the note for OCD customers who may want to know for sure.